config/aws/credentials

```diff
@@ -0,0 +1,3 @@
+[default]
+aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```

Comment:
so these are the keys that goaws will accept on the Drupal VM? No usage elsewhere?